packages/dotnet-6.vm/dotnet-6.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>dotnet-6.vm</id>
+    <version>0.0.0.20231224</version>
+    <authors>Microsoft</authors>
+    <description>Metapackage for .NET6 to ensure all packages use the same version.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="dotnet-6.0-desktopruntime" version="[6.0.25]" />
```

I suggest we use the following syntax to avoid that the automation updates the dependency, as that could break the tools using this metapackage:

```suggestion
      <dependency id="dotnet-6.0-desktopruntime" version="(6.0.13, 6.0.25]" />
```

---

We could also restrict it more with `(6.0.24, 6.0.25]` but I think allowing a bigger range of minor versions may prevent installing several versions.

---

Done